### PR TITLE
Repro/column titles

### PR DESCRIPTION
**Pull request by Opalevanescence**
Reproduction of Issue #11452

### CODE REVIEW

File: frontend/test/metabase/scenarios/question/notebook.cy.spec.js [hunk 1]
```diff
@@ -97,6 +98,25 @@ describe("scenarios > question > notebook", () => {
...
+      cy.get(".Icon-notebook").click();
+      cy.findByText("Join data").click();
+      cy.findByText("Reviews").click();
+      cy.findByText("Product ID").click();
+      popover().within(() => {
+        cy.findByText("Product ID").click();
+      });
+
+      // Column titles should not include 'Product'
+      cy.get(".QueryBuilder .Icon-sum").click();
```
Comment (paulrosenzweig) [inline frontend/test/metabase/scenarios/question/notebook.cy.spec.js:113]: Would "Summarize" work too?

File: frontend/test/metabase/scenarios/question/notebook.cy.spec.js [hunk 2]
```diff
@@ -97,6 +98,25 @@ describe("scenarios > question > notebook", () => {
...
+      cy.findByText("Pick a column to group by").click();
+      cy.get(".ReactVirtualized__List").within(() => {
```
Comment (paulrosenzweig) [inline frontend/test/metabase/scenarios/question/notebook.cy.spec.js:115]: Could this use `popover`? There might be multiple virtualized lists on the page.

File: frontend/test/metabase/scenarios/question/notebook.cy.spec.js [hunk 3]
```diff
@@ -97,6 +98,25 @@ describe("scenarios > question > notebook", () => {
...
+        cy.findAllByText("Product").should("not.exist");
```
Comment (paulrosenzweig) [inline frontend/test/metabase/scenarios/question/notebook.cy.spec.js:116]: I think we _should_ find "Product" in the list. My understanding is that the first appearance of "Product" below should be "Review", but the second is correct.
image

**Review by paulrosenzweig — CHANGES_REQUESTED** [review]

File: frontend/test/metabase/scenarios/question/notebook.cy.spec.js (hunk 1 above)
Comment (Opalevanescence) [reply frontend/test/metabase/scenarios/question/notebook.cy.spec.js:113]: Yeah, that works too

File: frontend/test/metabase/scenarios/question/notebook.cy.spec.js (hunk 2 above)
Comment (Opalevanescence) [reply frontend/test/metabase/scenarios/question/notebook.cy.spec.js:115]: Yeah, that also works :)

File: frontend/test/metabase/scenarios/question/notebook.cy.spec.js (hunk 3 above)
Comment (Opalevanescence) [reply frontend/test/metabase/scenarios/question/notebook.cy.spec.js:116]: Gotcha, I think I misunderstood; I'll correct that.

**Comment by nemanjaglumac** [comment]
Not quite sure what was going on here. Instead of having only reproduction code, this PR was also touching `spinner.cy.spec.js` (which is already in the master) o.O

Anyway, it was much easier to extract repro code than to deal with merge conflicts and to solve this puzzle.
Closing in favor of #13317.

**Pull request closed without merging**